boards/arm/mimxrt685_evk/mimxrt685_evk_cm33_defconfig

```diff
@@ -17,3 +17,6 @@ CONFIG_TRUSTED_EXECUTION_SECURE=y
 
 CONFIG_ARM_MPU=y
 CONFIG_HW_STACK_PROTECTION=y
+
+# Enable PMIC initialization for 1.8V SDHC support
+CONFIG_SDMMC_USDHC_PMIC_1_8_VOL=y
```

Comment:
This should move to the Kconfig.defconfig file with a depends on `SDMMC_USDHC` config